pravega/flink-connectors
# [Issue-705] Implement sink metrics

**Change log description**
With #460 , the new sink interface needs to expose certain metrics.
We only add `numRecordsOut` and `numRecordsOutError` metrics. `numBytesOut` is not available because Pravega itself does not have metrics on writer side.

**Purpose of the change**
Fix #705 

**What the code does**
Implement `numRecordsOut` for both event writer and transaction writer, implement `numRecordsOutError` for event writer.

**How to verify it**
`./gradlew clean build` should pass
Such metrics should be able to view through Flink UI or Flink RSET API
